frontend/src/features/admin-form/common/AdminFormLayout.tsx

```diff
@@ -1,12 +1,29 @@
-import { Outlet } from 'react-router-dom'
+import { Outlet, useParams } from 'react-router-dom'
 import { Flex } from '@chakra-ui/react'
+import { get } from 'lodash'
+
+import AdminForbidden403 from '~pages/AdminForbidden403'
```

Comment:
small nit but might be better to name these after the purpose of the page rather than coupling to a specific status code, e.g. `NoPermissionToAdminForm` (not the best name but you get the idea)

Reply:
will change to AdminForbiddenErrorPage?